docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -197,7 +197,7 @@ To start a Keycloak Server you can use Docker and just run the following command
 
 [source,bash,subs=attributes+]
 ----
-docker run --name keycloak -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -p 8180:8080 -p 8543:8443 {keycloak-docker-image}
+docker run --name keycloak -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -p 8180:8080 -p 8543:8443 quay.io/keycloak/keycloak:{keycloak version}
```

Comment:
I'm not sure how it is an improvement? Also `{keycloak version}` is not replaced in the documentation, is it expected? I would have expected it to be a proper attribute.

Reply:
@sberyozkin ^

Reply:
HI @gsmet, , do you mean how `{keycloak-docker-image}` is better than `quay.io/keycloak/keycloak:{keycloak version}` ? 

I think new users won't necessarily know that it needs to be replaced with `quay.io/keycloak:someversion` - but I've actually found myself as well going to `build/parent/pom.xml` copying `quay.io/keycloak/keycloak:{keycloak version}` all the time when I need to start Keycloak manually. So IMHO with the new update it is easier for the users to get the right image - example, I just know `{keycloak.version}` is `15.0.2` so I copy `quay.io/keycloak/` and replace the version template var only, while the new users can copy `quay.io/keycloak/` to the browser and go there and check the tag.

> Also {keycloak version} is not replaced in the documentation, is it expected? I would have expected it to be a proper attribute.

I don't know how to sync the Keycloak properties from either `build-parent/pom.xml` or `bom/application/pom.xml`, is it what you meant, to have it reflecting the current version supported by Quarkus ?

Reply:
Oh I see, Guillaume, @gsmet, you probably thought `{keycloak-docker-image}` was replaced in the published docs, it is not: https://quarkus.io/guides/security-openid-connect#starting-and-configuring-the-keycloak-server, so here I usually copy that command and start replacing it manually.
If it can be fixed such that the actual property used in Quarkus pom is also used to replace `{keycloak-docker-image}` in the docs then it would be nice - not sure it is possible.

But at least may be I can replace `{keycloak version}` with `{keycloak.version}` to use the same property name as we use in our poms, let me know if you prefer it